<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanProgram</class>
 <widget class="QDialog" name="ScanProgram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="select_btn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>960</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>File Select</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_zone">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1160</width>
     <height>900</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white</string>
   </property>
   <property name="text">
    <string>Please Select Image File</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="scan_result">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>975</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Scan Result :</string>
   </property>
  </widget>
  <widget class="QLabel" name="score_label">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>1025</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Scan Score :</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="scan_txt">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>970</y>
     <width>256</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="score_txt">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>1020</y>
     <width>256</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="transmit_btn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>1030</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Transmit</string>
   </property>
  </widget>
  <widget class="QLabel" name="send_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>1100</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>send image data for scan</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="connect_btn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>1070</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="port_num_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>975</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Port :</string>
   </property>
  </widget>
  <widget class="QLabel" name="ip_num_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>1025</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> IP :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disconnect_btn">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>1120</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: yellow;
background-color: #ff0000;
</string>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="port_num">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>970</y>
     <width>256</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;input port number&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="ip_num">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>1020</y>
     <width>256</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;input ip number&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
